mk/spksrc.service.installer.dsm5

```diff
@@ -35,7 +35,12 @@ call_func ()
     FUNC=$1
     if type "${FUNC}" 2>/dev/null | grep -q 'function' 2>/dev/null; then
         install_log "Begin ${FUNC}"
-        eval ${FUNC} 2>&1 | install_log
+        ARG=$2
+        if [ -z "${ARG}" ]; then
+            eval ${FUNC} 2>&1 | install_log
+        else
+            eval ${FUNC} ${ARG} 2>&1 | install_log
+        fi
```

Comment:
Good catch.  You made a comment pertaining to DSM5 vs 6-7 behavior.  A comment on this matter would be nice.

Opened question: why limiting to 1 argument?  Couldn't we catch all remaining arguments and assign it to `$ARG` ?

Reply:
It's the same information in https://github.com/SynoCommunity/spksrc/pull/5592#issuecomment-1426532254. For DSM 5 versus DSM 6/7, the argument for the log file was not yet introduced. I was trying to minimise the number of changes to the framework by catching the single argument passed from the `call_func` rather than updating it to match the DSM 6/7 logic.